### ISSUE TITLE
Incompatible with latest web-ext release (5.0.0)

### ISSUE THREAD

**Issue by GregJPreece** [mmktomato/parcel-plugin-web-ext-tool#5]
When trying to use the plugin with the newly released web-ext 5.0, the plugin fails with the following error:

```
> opentitles-browser@0.1.0 watch /path/to/my/project/
> parcel watch ./src/test.ts --public-url ./

✨  Built in 897ms.
(node:27221) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'cmd' of undefined
    at Bundler.<anonymous> (/path/to/my/project/node_modules/parcel-plugin-web-ext-tool/src/index.js:19:27)
(node:27221) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:27221) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
^C
```

Downgrading to web-ext 4.3.0 fixes the issue. Looking at [the release notes for web-ext 5.0.0](https://github.com/mozilla/web-ext/releases/tag/5.0.0) it seems that their exports have changed.

**Comment by mmktomato** [comment]
Thank you for reporting. I'll check it.

**Comment by mmktomato** [comment]
@GregJPreece I published 0.2.0. It requires `web-ext` >= 5.x.x. Could you check it?

**Comment by GregJPreece** [comment]
Works perfectly, thank you for such a swift fix!
